docs/source/operations/guide-use-armada.rst

```diff
@@ -0,0 +1,56 @@
+Armada - Using Armada
+=====================
+
+Prequesits
+----------
+
+Kubernetes Cluster
+
+Tiller Service `here <http://github.com/kubernetes/helm>`_
+
+Armada.yaml `here <guide-build-armada-yaml.rst>`
+
+
+Usage
+-----
+
+1. Pull the Armada Docker Images:
+
+.. code:: bash
+
+    docker pull quay.io/attcomdev/armada:latest
+
+
+2. Run Armada docker container
+
+.. note::
+
+    Make sure to mount your kubeconfig into ``/root/.kube/config`` in
+    the container
+
+.. code:: bash
+
+    docker run -d --net host -p 8000:8000 --name armada -v ~/.kube/config:/root/.kube/config -v $(pwd)/armadayamls/:/armadayaml quay.io/attcomdev/armada:latest
+
+
+3. Check that tiller is Available
+
+.. code:: bash
+
+    docker exec -it armada armada tiller --status
+
+
+4. If tiller is up then we can start deploying our armadayamls
+
+.. code:: bash
+
+    docker exec -it armada armada apply /armadayaml/file-name [ --debug-logging ]
```

Comment:
can you explain where these come from? additionally, it would be good to have an example of the expected output from an example YAML.